fe/fe-core/src/main/java/com/starrocks/scheduler/TaskRun.java

```diff
@@ -177,6 +177,8 @@ public boolean executeTaskRun() throws Exception {
                 } catch (DdlException e) {
```

Comment:
is it more safer to use `checkSystemVariableExist` to check var exist before modify ?

Reply:
it's much more complicated that a vanilla check, since there're some other properties with the variable